test/integration/integration_test.go

```diff
@@ -66,9 +66,23 @@ func TestMain(m *testing.M) {
 	}
 	runner := util.MinishiftRunner{CommandPath: minishiftBinary}
 	if runner.IsCDK() {
-		godogTags += "~minishift-only"
+		godogTags += "~minishift-only&&~b2d-only&&~centos-only"
 	} else {
-		godogTags += "~cdk-only"
+		switch os.Getenv("MINISHIFT_ISO_URL") {
+		case "b2d":
+			godogTags += "~cdk-only&&~centos-only&&~rhel-only"
```

Comment:
@hferentschik @praveenkumar If need to differentiate between centos and rhel is not most likely happen, then we can remove the rhel-only tag and handle it as centos. It is true that we can exclude some centos scenarios by adding minishift-only, this will prevent it from running in downstream and in fact will behave as rhel-only.